### PR TITLE
Added T4 weapon names.

### PR DESCRIPTION
**Pull request by efinst0rm**
This list should be correct with all the current world at war weapon names.

### CODE REVIEW

File: Application/DefaultSettings.json [hunk 1]
```diff
@@ -1115,6 +1115,59 @@
...
+	  "left_arm_upper": "Upper Left Arm",
+	  "left_arm_lower": "Lower Left Arm",
+	  "gl": "Rifle Grenade",
+	  "bigammo": "Round Drum",
+	  "scoped": "Sniper Scope",
+	  "telescopic": "Telescopic Sight",
+	  "aperture": "Aperture Sight",
+	  "flash": "Flash Hider",
+	  "silenced": "Silencer",
+	  "tabun_gas": "Tabun Gas",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1141]: So the way the parser works is it always splits the weapons with _ into two parts
tabun and gas. 
It also automatically converts it to title case so this one is not needed

File: Application/DefaultSettings.json [hunk 2]
```diff
@@ -1115,6 +1115,59 @@
...
+	  "signal_flare": "Signal Flare",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1142]: this one can be omitted as well

File: Application/DefaultSettings.json [hunk 3]
```diff
@@ -1115,6 +1115,59 @@
...
+	  "molotov": "Molotov Cocktail",
+	  "sticky_grenade": "N° 74 ST",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1144]: this should just be `sticky: N° 74 ST` due to same concept as previous comments

File: Application/DefaultSettings.json [hunk 4]
```diff
@@ -1115,6 +1115,59 @@
...
+	  "m2_flamethrower": "M2 Flamethrower",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1145]: `m2`

File: Application/DefaultSettings.json [hunk 5]
```diff
@@ -1115,6 +1115,59 @@
...
+	  "artillery": "Artillery Strike",
+	  "dog_bite": "Attack Dogs",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1147]: `dog`

**Comment by efinst0rm** [comment]
I updated everything!

File: Application/DefaultSettings.json [hunk 6]
```diff
@@ -1115,6 +1115,57 @@
...
+	  "left_arm_lower": "Lower Left Arm",
+	  "gl": "Rifle Grenade",
+	  "bigammo": "Round Drum",
+	  "scoped": "Sniper Scope",
+	  "telescopic": "Telescopic Sight",
+	  "aperture": "Aperture Sight",
+	  "flash": "Flash Hider",
+	  "silenced": "Silencer",
+	  "molotov": "Molotov Cocktail",
+	  "sticky_grenade": "N° 74 ST",
```
Comment (RaidMax) [inline Application/DefaultSettings.json:1142]: looks like this one is still underscored

Comment (efinst0rm) [reply Application/DefaultSettings.json]: Ok should be fixed. Now.

**Review by RaidMax — APPROVED** [review]

**Pull request merged**